src/Core/Geographic/Extent.js

```diff
@@ -112,6 +112,7 @@ Extent.prototype.as = function as(crs) {
         let east = -Infinity;
         let west = Infinity;
         for (let i = 0; i < cardinals.length; i++) {
+            cardinals[i]._internalStorageUnit = this._internalStorageUnit;
```

Comment:
It seems correct but would require a comment because it "looks" wrong (because we overwrite `cardinals[i]`  at the next line).

Reply:
ok, i have updated the code.